parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -101,10 +103,17 @@ impl NativeTokensMessage {
 				create_call_index,
 				set_metadata_call_index,
 			} => {
-				let owner = GlobalConsensusEthereumAccountConvertsFor::<[u8; 32]>::from_params(
-					&chain_id,
-					origin.as_fixed_bytes(),
-				);
+				let owner =
+					GlobalConsensusEthereumAccountConvertsFor::<[u8; 32]>::convert_location(
+						&MultiLocation::new(
+							2,
+							(
+								GlobalConsensus(Ethereum { chain_id }),
+								AccountKey20 { network: None, key: *origin.as_fixed_bytes() },
```

Comment:
When would we set a network id in an account key?

Reply:
Guess already set in `GlobalConsensus(network)` above so can just ignore here.

Reply:
An example of where we "could" use network id is if we are specifying an ERC20 token contract on an EVM based parachain and potentially transferring it over the bridge and we would want a way to specify its Network as `Kusama` so we could handle it specially in some way. But I say "could" here because it's hypothetical.

If you look at some places in the source code, we often ignore network id. Such as [here](https://github.com/paritytech/polkadot/blame/master/xcm/xcm-builder/src/location_conversion.rs#L67-L85).
There are also some places where it makes sense to validate it if present like [here](https://github.com/Snowfork/snowbridge/blob/main/parachain/primitives/router/src/outbound/mod.rs#L272-L274).

In this case yrong is correct in that it is set in GlobalConsensus so we can set it as None on the key itself.